runtime/vm/jvminit.c

```diff
@@ -4218,6 +4218,18 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		} else if (flightRecorder < noFlightRecorder) {
 			vm->extendedRuntimeFlags2 &= ~(UDATA)J9_EXTENDED_RUNTIME2_JFR_ENABLED;
 		}
+
+		IDATA jfrOptionIndex = FIND_AND_CONSUME_VMARG(OPTIONAL_LIST_MATCH_USING_EQUALS, VMOPT_XXSTARTFLIGHTRECORDING, NULL);
```

Comment:
this code should be wrapped in `#if defined(J9VM_OPT_JFR)`

Reply:
Yeah, Code is already within the `#if defined(J9VM_OPT_JFR)`